### DISCUSSION TITLE
Possible AKAO velicoty fix?

### DISCUSSION THREAD

**Discussion by InanimateCrbnRod** [vgmtrans/vgmtrans#745] [Music formats]
Forgive me, I am not a coder, just a video game music enthusiast. Currently VGMTrans doesn't seem to extract midi velocity data from FF7 Akao files. But it seems to get most of the other info, and it appears even bank information which is super handy. 

A few years back a developer named valleybell made this https://github.com/ValleyBell/MidiConverters/blob/master/AKAO2MID.bas
which doesn't get any of the bank information but does indeed seem to get midi velocity data. Is there anyway to take the info that akao2mid is finding and incorporate it into VGMTrans? then we might actually be able to have functioning midi replicas of the PSX FF7 score, a big boon for ff7 music modders. Hope this helps, and either way thank you and keep up the good work!

**Comment by loveemu** [comment]
Instead of AKAO2MID mapping vcmd A3 to velocity, VGMTrans maps it to CC7 (volume). In some driver versions, this volume control can be changed by the fader while the note is being played, so it makes sense to map it to a control change instead of velocity.

**Comment by InanimateCrbnRod** [comment]
Oh I see so it's not pulling different data than vgmtrans it's just putting it on cc7 instead.
